components/zerotier/actions/get-network/get-network.mjs

```diff
@@ -3,9 +3,9 @@ import zerotier from "../../zerotier.app.mjs";
 export default {
   key: "zerotier-get-network",
   type: "action",
-  version: "0.0.1",
+  version: "0.0.2",
   name: "Get Network",
-  description: "Get a specific network. [See docs here](https://docs.zerotier.com/central/v1/#operation/getNetworkByID)",
+  description: "Get a specific network. Returns a single network.",
```

Comment:
Can you plese bring back again all the doc references in the description of each component? Thanks!
```suggestion
  description: "Get a specific network. Returns a single network. [See docs here](https://docs.zerotier.com/central/v1/#operation/getNetworkByID)",
```

Reply:
@jcortes done